js/data.js

```diff
@@ -1,8 +1,21 @@
 /* exported data */
+window.addEventListener('beforeunload', saveToLocalStorage);
 
 var data = {
   view: 'entry-form',
   entries: [],
   editing: null,
   nextEntryId: 1
 };
+
+var previousDataJSON = localStorage.getItem('journal-entry');
+
+if (previousDataJSON !== null) {
+  data = JSON.parse(previousDataJSON);
+}
+
+function saveToLocalStorage(event) {
+  var dataJSON = JSON.stringify(data);
+  localStorage.setItem('journal-entry', dataJSON);
+}
+data.nextEntryId++;
```

Comment:
This should be removed 👀 

This should only be incremented when a new entry is submitted